plugins/woocommerce/includes/class-wc-session-handler.php

```diff
@@ -117,7 +117,7 @@ private function init_session() {
 	 * @return bool
 	 */
 	private function init_session_from_request() {
-		$session_token = wc_clean( wp_unslash( $_GET['session'] ?? '' ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$session_token = is_string( $_GET['session'] ?? '' ) ? wc_clean( wp_unslash( $_GET['session'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I wonder if this wouldn't make more sense, to handle the case when `$_GET['session'] ` is not set slightly better?

```suggestion
		$session_token = is_string( $_GET['session'] ?? '' ) ? wc_clean( wp_unslash( $_GET['session'] ?? '' ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Reply:
Good catch!